todo.js

```diff
@@ -0,0 +1,58 @@
+import { v4 as uuidv4 } from 'uuid'
+
+export class Todo {
+    #id
+    #isDone
+    #text
+
+    constructor(text) {
+        this.#id = uuidv4()
+        this.#isDone = false
+        this.#text = text
+    }
+
+    getId() {
+        return this.#id
+    }
+
+    getIsDone() {
+        return this.#isDone
+    }
+
+    toggleIsDone() {
+        this.#isDone = !this.#isDone
+    }
+
+    setText(value) {
+        this.#text = value
+    }
+
+    render(onClick, createSpanBtns) {
+        const newDiv = document.createElement('div')
+        const newListItem = this.#createNewListItem(onClick)
+        const newSpan = createSpanBtns(this.#id)
```

Comment:
hmm we need to figure out how to include `createSpanBtns` in the class, because this is still external dependency